README.md

```diff
@@ -422,6 +422,7 @@ task_definition:
   services:
     <service_name>:
       essential: boolean
+      repository_credentials: string
```

Comment:
`repository_credentials` is actually an object with a `credentials_param` field that takes a string:
https://github.com/aws/amazon-ecs-cli/commit/1f8c534d43dc02c062b0db76f8e7f753e132a4a2